mica-core/src/main/resources/taxonomies/mica-dataset.yml

```diff
@@ -64,10 +64,10 @@
   "description":
     "en": "Type of dataset (e.g. data collection or harmonization)"
   "terms":
-  - "name": "StudyDataset"
+  - "name": "CollectedDataset"
```

Comment:
Are you sure of that? this is the class name, which is not supposed to have changed.